### ISSUE TITLE
Window heating with climatization

### ISSUE THREAD

**Issue by gerliczky** [skodaconnect/homeassistant-myskoda#281]
In the Skoda connect integration there was a switch to optionally to turn on window heating, when electric climatization is turned on. Is it possible to implemet that also here?

**Comment by prvakt** [comment]
can you provide screenshot? Checking the old integration but window heater is separated from the climatisation switch
![image](https://github.com/user-attachments/assets/b80c3b43-371e-4013-a14c-71182267cc75)

**Comment by gerliczky** [comment]
I marked the two switches related to window heating.
![image](https://github.com/user-attachments/assets/85aefad6-81e8-40eb-92fe-ddb1ca9c7f48)

In the My Skoda app you can enable automatic windows heating with climatization, if you click on climate control, then the three dits on the upper left corner and there you can find a switch under Automatic intelligent heating, called Window heating.

**Comment by prvakt** [comment]
ok this is not available in my Superb iV.. I can additionally choose only to enable or disable AC from battery if charging cable is unplugged.. Would it be possible to generate fixtures for your car?

**Comment by gerliczky** [comment]
If you give me instructions, I can assist.

**Comment by prvakt** [comment]
ideally by running myskoda[cli] (https://github.com/skodaconnect/myskoda?tab=readme-ov-file#as-cli). I'm not sure how much experience you have, but its possible to use ubuntu docker image with python installed  
```
install ubuntu:latest docker image, then
1) apt-get update
2) apt-get install -y python3 python3-pip python3-venv (you will be prompted to enter your timnezone info)
3) python3 -m venv myenv 
4) source myenv/bin/activate
4) pip install myskoda[cli]
5) myskoda --help (and just follow the command instructions to generate fixtures)
 should take max 5 minutes and you are ready to test 😉
```
if its too complicated, you can enable debug logging and API tracing in myskoda integration and provide anonymized responses from the log - especially for the air-conditioning

**Comment by WebSpider** [comment]
I have the features in my Enyaq. Just don't have the API command to enable/disable the seat heating, window heating, etc.

**Comment by gerliczky** [comment]
I attached HA debug log. I hope it helps.
[enyaq.txt](https://github.com/user-attachments/files/17781101/enyaq.txt)

**Comment by prvakt** [comment]
probably found the endpoint and body configuration in decompiled apk

`api/v2/air-conditioning/{vin}/settings/windows-heating`
Body
```
{
    "windowHeatingEnabled" = True/False
}
```
will implement it once I'm finished with the AuxiliaryHeater

**Comment by prvakt** [comment]
#355 will add Windows Heating button with AC ;)

**Comment by WebSpider** [comment]
This has been released with v1.12.0
